filesystems/gcs/src/main/scala/cromwell/filesystems/gcs/batch/GcsBatchCommandBuilder.scala

```diff
@@ -1,31 +1,39 @@
 package cromwell.filesystems.gcs.batch
 
 import cromwell.core.io._
+import cromwell.core.path.BetterFileMethods.OpenOptions
 import cromwell.core.path.Path
 import cromwell.filesystems.gcs.GcsPath
+import cromwell.filesystems.oss.OssPath
+import cromwell.filesystems.oss._
 
 trait GcsBatchCommandBuilder extends DefaultIoCommandBuilder {
```

Comment:
I believe the changes int his file are unnecessary but we've asked @Horneth to chime in

Reply:
Yeah, @crisish can you explain why you're piggybacking on GCS here?

Reply:
I removed it.